Tests/SwiftDocCTests/Model/RenderHierarchyTranslatorTests.swift

```diff
@@ -125,4 +127,104 @@ class RenderHierarchyTranslatorTests: XCTestCase {
+            for reference: ResolvedTopicReference,
+            expectedSwiftPaths: [String]?,
+            expectedObjectiveCPaths: [String]?,
+            file: StaticString = #filePath,
+            line: UInt = #line
+        ) throws {
+            let documentationNode = try context.entity(with: reference)
+            var translator = RenderNodeTranslator(context: context, bundle: bundle, identifier: reference)
+            let renderNode = try XCTUnwrap(translator.visit(documentationNode.semantic) as? RenderNode, file: file, line: line)
+            
+            if let expectedSwiftPaths {
+                guard case .reference(let defaultHierarchy) = renderNode.hierarchyVariants.defaultValue else {
+                    XCTFail("Unexpectedly found `.tutorials` main hierarchy for symbol", file: file, line: line)
+                    return
+                }
+                XCTAssertEqual(defaultHierarchy.paths.count, 1, "Unexpectedly found \(defaultHierarchy.paths.count) symbol paths", file: file, line: line)
+                XCTAssertEqual(defaultHierarchy.paths.first, expectedSwiftPaths, file: file, line: line)
+            } else {
+                XCTAssertNil(renderNode.hierarchyVariants.defaultValue, "Unexpectedly found main hierarchy", file: file, line: line)
+            }
+                
+            if let expectedObjectiveCPaths {
+                let variants = try XCTUnwrap(renderNode.hierarchyVariants.variants.first, file: file, line: line)
+                let patch = try XCTUnwrap(variants.patch.first, file: file, line: line)
+                guard case .replace(value: .reference(let variantHierarchy)) = patch else {
+                    XCTFail("Unexpectedly found `.tutorials` variant hierarchy for symbol", file: file, line: line)
+                    return
+                }
+                XCTAssertEqual(variantHierarchy.paths.count, 1, "Unexpectedly found \(variantHierarchy.paths.count) symbol paths", file: file, line: line)
+                XCTAssertEqual(variantHierarchy.paths.first, expectedObjectiveCPaths, file: file, line: line)
+            } else {
+                XCTAssertNil(renderNode.hierarchyVariants.variants.first, "Unexpectedly found variant hierarchy", file: file, line: line)
+            }
+        }
+        
+        // typedef struct {
+        //     CGPoint center;
+        //     CGFloat radius;
+        // } TLACircle NS_SWIFT_NAME(Circle);
+        try assertExpectedHierarchies(
+            for: moduleReference.appendingPath("Circle/center"),
+            expectedSwiftPaths: [
+                "doc://GeometricalShapes/documentation/GeometricalShapes",
+                "doc://GeometricalShapes/documentation/GeometricalShapes/Circle",
+            ],
+            expectedObjectiveCPaths: [
+                "doc://GeometricalShapes/documentation/GeometricalShapes",
+                "doc://GeometricalShapes/documentation/GeometricalShapes/Circle", // named TLACircle in Objective-C
```

Comment:
Why is the Objective-C breadcrumb the same as the Swift breadcrumb here?

Reply:
Because the `center` property is a member of the `TLACircle` structure (see the comment above the assertion)

Reply:
Since the symbol exist for both Swift and Objective-C it only has a single reference (which uses the Swift path)